core-web/libs/portlets/dot-ema/src/lib/feature/store/dot-ema.store.ts

```diff
@@ -37,7 +46,25 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
     readonly language_id$: Observable<string> = this.select((state) => state.language_id);
     readonly url$: Observable<string> = this.select((state) => state.url);
     readonly pageTitle$: Observable<string> = this.select((state) => state.editor.page.title);
+    readonly editIframeURL$: Observable<string> = this.select((state) => state.dialogIframeURL);
 
+    readonly vm$ = this.select((state) => {
+        return {
+            iframeUrl: `http://localhost:3000/${state.url}?language_id=${state.language_id}`,
```

Comment:
Shouldn't this come from the`ema` APP Configuration we have in dotCMS? @fmontes @zJaaal 

<img width="1469" alt="dotCMS - EMA APP Configuration" src="https://github.com/dotCMS/core/assets/72418962/f2ec6617-c983-41c0-9167-c2049504252f">

Reply:
Coming soon.

Reply:
Got it!

Reply:
We don't have an available endpoint to get this data from there. So for now we are hardcoding it for development purpose